### PR TITLE
add pool id as part of 'mc admin info' output

### PR DESCRIPTION
**Pull request by harshavardhana**
This also fixes `mc` output when it incorrectly displays offline disks.

### CODE REVIEW

**Review by klauspost — APPROVED** [review]
LGTM

File: cmd/admin-info.go [hunk 1]
```diff
@@ -230,12 +226,19 @@ func (u clusterStruct) String() (msg string) {
...
-		msg += fmt.Sprintf("%s online, %s offline\n",
-			english.Plural(totalOnlineDisksCluster, "drive", ""),
-			english.Plural(totalOfflineDisksCluster, "drive", ""))
+		bkInfo, ok := u.Info.Backend.(madmin.ErasureBackend)
+		if ok {
+			msg += fmt.Sprintf("%s online, %s offline\n",
+				english.Plural(bkInfo.OnlineDisks, "drive", ""),
+				english.Plural(bkInfo.OfflineDisks, "drive", ""))
+		} else {
+			msg += fmt.Sprintf("%s online, %s offline\n",
```
Comment (vadmeste) [inline cmd/admin-info.go:238]: Printing disks offline/online does not work for gateway. I think we don't have to print anything if this is not erasure.

**Review by vadmeste — COMMENTED** [review]
One comment

File: cmd/admin-info.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/admin-info.go:238]: How does gateway come into picture here @vadmeste 

This code never prints in Gateway. Did you test it with gateway?

Comment (harshavardhana) [reply cmd/admin-info.go:238]: done fixed @vadmeste

**Review by vadmeste — APPROVED** [review]
LGTM

**Pull request merged**